src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -163,16 +160,11 @@ internal sealed partial class ConsoleHost
                     hostException = e;
                 }
 
-                if (args == null)
```

Comment:
We should probably leave the null check in the code.

Reply:
Is c# convention that args is never null?